client/src/components/downloadPacket.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Button} from '@trussworks/react-uswds';
+import * as styles from './downloadPacket.module.scss';
+// @ts-ignore
+import downloadIcon from '/node_modules/uswds/dist/img/usa-icons/file_download.svg';
+
+const DownloadPacket = () => {
+  const cbgFileURL = 'https://usds-geoplatform-justice40-website.s3.amazonaws.com/data/score/csv/usa.csv';
+  return (
+    <div className={styles.downloadBoxContainer}>
+      <div className={styles.downloadBox}>
+        <div className={styles.downloadBoxTextBox}>
+          <div className={styles.downloadBoxTitle}>Just Progress Packet</div>
+          <div className={styles.downloadBoxText}>This downloadable packet includes the list of Just Progress
+          prioritized communities (30,021 census block groups) and 18 datasets.
```

Comment:
Ah, just saw these hard-coded values - maybe we could parameterize this somehow? We could have it pull from graphql potentially and store this as we build? This could be a followup ticket also

Reply:
I was thinking this also. If the content was coming in from another source (CMS), we could have product/design edit it as well. Might be worth looking into another ticket for this.